src/cudamatrix/cu-matrix.cc

```diff
@@ -2540,33 +2572,53 @@ void CuMatrixBase<Real>::Lookup(const std::vector<Int32Pair> &indices,
   // Checks the pointer.
   KALDI_ASSERT(output != NULL);
```

Comment:
Please write
if (indices.size() == 0) return;
before this line.  output may be NULL if indices.size() is zero.